components/bunnygrabber.py

```diff
@@ -8,12 +8,21 @@ class BunnyGrabber:
     direction = 0
 
     def set_backward(self):
+        """
+            Sets motor direction to backwards.
+        """
         self.direction = -1
 
     def set_forward(self):
+        """ Sets motor direction to forwards.
+        """
         self.direction = 1
 
     def execute(self):
+        """ Moves the motor based on the direction variable.
```

Comment:
Anyone working with this code should already know what a components `execute` method does, or they should look it up. This behavior is essentially what every single execute does. This docstring should be split up and added as standard comments to the actual code within the `execute` method, `execute` doesn't need a docstring.